tests/tests/starknet/secp256.rs

```diff
@@ -533,65 +538,6 @@ fn secp256k1_add() {
         },
     );
 
```

Comment:
You removed the asserts that were here. Are we checking the values in a different place?